classes/Product.php

```diff
@@ -4455,6 +4455,8 @@ public static function getProductProperties($id_lang, $row, Context $context = n
             $row['unit_price_ratio'] = $row['price_tax_exc'] / $unitPrice;
         }
 
+        $row['unit_price'] = ($row['unit_price_ratio'] != 0  ? $row['price'] / $row['unit_price_ratio'] : 0);
```

Comment:
It's formatted in the `ProductPresenter` according to the locale settings. The more info we have, the better the presentation will be.